rooms/rmBeta.room.gmx

```diff
@@ -1377,6 +1375,14 @@
     <instance objName="objTellySpawner" x="2688" y="240" name="inst_E1366C4C" locked="0" code="" scaleX="1" scaleY="1" colour="4294967295" rotation="0"/>
     <instance objName="objTellySpawner" x="2720" y="240" name="inst_FF66FDA3" locked="0" code="" scaleX="1" scaleY="1" colour="4294967295" rotation="0"/>
     <instance objName="objTellySpawner" x="2752" y="240" name="inst_6B740314" locked="0" code="" scaleX="1" scaleY="1" colour="4294967295" rotation="0"/>
+    <instance objName="objBBitter" x="352" y="320" name="inst_9F3E4F92" locked="0" code="" scaleX="1" scaleY="1" colour="4294967295" rotation="0"/>
+    <instance objName="objSniperJoe" x="1648" y="352" name="inst_2C094D07" locked="0" code="" scaleX="1" scaleY="1" colour="4294967295" rotation="0"/>
```

Comment:
I put it that way so a player doesn't get blind-sided.

I'll add a small patch of elevated ground for the Joe on the ground.